p2p/starknet/starknet_test.go

```diff
@@ -287,3 +457,52 @@ func randFelt(t *testing.T) *felt.Felt {
 
 	return f
 }
+
+func fillFelts[T any](t *testing.T, i T) T {
```

Comment:
I think this should be moved to `utils` package along with `randFelt` and `feltSlice` as they can be useful in other tests. However, we can do it in another pr.